Apps/W1/Shopify/app/src/Products/Codeunits/ShpfyCreateProduct.Codeunit.al

```diff
@@ -256,6 +184,48 @@ codeunit 30174 "Shpfy Create Product"
             exit(ItemReferenceMgt.GetItemReference(ItemNo, VariantCode, UoM, "Item Reference Type"::Vendor, Item."Vendor No."));
     end;
 
+    local procedure GetVariantSKU(BarCode: Text[50]; ItemNo: Text[20]; VariantCode: Text[10]; VendorItemNo: Text[50]): Text[50]
+    begin
+        case Shop."SKU Mapping" of
+            Shop."SKU Mapping"::"Bar Code":
+                exit(BarCode);
+            Shop."SKU Mapping"::"Item No.":
+                exit(ItemNo);
+            Shop."SKU Mapping"::"Variant Code":
+                if VariantCode <> '' then
+                    exit(VariantCode);
+            Shop."SKU Mapping"::"Item No. + Variant Code":
+                if VariantCode <> '' then
+                    exit(ItemNo + Shop."SKU Field Separator" + VariantCode)
+                else
+                    exit(ItemNo);
+            Shop."SKU Mapping"::"Vendor Item No.":
+                exit(VendorItemNo);
+        end;
+    end;
+
+    /// <summary>
+    /// Creates a temporary Shopify variant with information from an item.
+    /// </summary>
+    /// <param name="Item">The item to create the variant from.</param>
+    /// <param name="TempShopifyVariant">The temporary Shopify variant record set where the variant will be inserted.</param>
+    internal procedure CreateTempShopifyVariantFromItem(Item: Record Item; var TempShopifyVariant: Record "Shpfy Variant" temporary)
+    begin
+        Clear(TempShopifyVariant);
+        TempShopifyVariant."Available For Sales" := true;
+        TempShopifyVariant.Barcode := CopyStr(GetBarcode(Item."No.", '', Item."Sales Unit of Measure"), 1, MaxStrLen(TempShopifyVariant.Barcode));
+        ProductPriceCalc.CalcPrice(Item, '', Item."Sales Unit of Measure", TempShopifyVariant."Unit Cost", TempShopifyVariant.Price, TempShopifyVariant."Compare at Price");
+        TempShopifyVariant.Title := ''; // Title will be assigned to "Default Title" in Shopify as no Options are set.
```

Comment:
Previously, this was set to ItemVariant.Description, but ItemVariant was never retrieved/Initialized in this code path, so it was effectivelly always empty. For clarity, this could also be "Default Title", as that's what the title is going to be in Shopify.